test/entities.go

```diff
@@ -142,26 +142,29 @@ func (g *Blocks) New() *flow.Block {
 }
 
 type BlockHeaders struct {
-	count    int
-	ids      *Identifiers
-	lastTime time.Time
+	count     int
+	ids       *Identifiers
+	startTime time.Time
 }
 
 func BlockHeaderGenerator() *BlockHeaders {
+	startTime, _ := time.Parse(time.RFC3339, "2020-06-04T15:43:21+00:00")
```

Comment:
This is great, just curious: Is there a particular reason why the test changed? Is it to make it deterministic?

Reply:
Yep! All of the other entity generators are deterministic in that they produce the exact same sequence of entities on each run. I realized that `time.Now` breaks determinism for the `Blocks` and `BlockHeaders` generators, so I made this change to make them consistent with the rest. It's not a strict requirement but I want to preserve consistency. :)

Thanks for the reviews BTW 🙇